WcaOnRails/app/webpacker/components/StaticPages/Logo.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { sanitize } from 'dompurify';
+import I18n from '../../lib/i18n';
+
+/**
+ * @returns {JSX.Element}
+ * @constructor
+ */
+function Logo({ title }) {
+  return (
+    <>
+      <h1>{title}</h1>
+      {I18n.t('logo.paragraphs').map((paragraph, index) => paragraph && (
+        <p
+          // eslint-disable-next-line react/no-array-index-key
+          key={`logo-paragraphs-${index}`}
+          // eslint-disable-next-line react/no-danger
+          dangerouslySetInnerHTML={{ __html: sanitize(paragraph) }}
```

Comment:
We have a component called `I18nHTMLTranslate` which is made precisely to avoid manual inner HTML overrides like this. Something along the lines of
```jsx
{I18n.t('logo.paragraphs').map((paragraph, index) => paragraph && (
    <I18nHTMLTranslate
      // eslint-disable-next-line react/no-array-index-key
      key={`logo-paragraphs-${index}`}
      i18nKey={'logo.paragraphs-${index+1}'}
    />
))}
```

should do the trick.

Reply:
Ahh ok thanks, I didn't realise you could index in the i18n key like that.

Reply:
Marking this as Resolved as per the other PR that I just merged.
Don't forget to rebase this PR and delete the (now redundant) CSS file!